docs/general/manage-tokens/holding-rose-tokens/ledger-wallet.md

```diff
@@ -40,15 +47,36 @@ You may see both the _Oasis_ app and the _OasisVal_ app come up in the search re
 
 :::
 
+
+
 Once installed, the Oasis app will use the [BIP 39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) mnemonic seed stored on your Ledger wallet to generate the private & public key pairs for your accounts.
 
 ## Manage Your Tokens
 
 ### Using Oasis Wallets
 
-This is a simpler option since it allows you to connect to your Ledger wallet via a web browser app or a web browser extension.
+This is a simpler option since it allows you to connect to your Ledger wallet via our official wallets. In our example we will use Oasis Wallet - Web. For how to use our official wallets please use instructions in the [Oasis Wallets](../oasis-wallets/README.mdx) doc.
```

Comment:
"This is a simpler option since it allows you to connect to your Ledger wallet via our official wallets." doesn't really explain why it is simpler, like it did before.

Reply:
"This is a simpler option since it allows you to connect to your Ledger wallet via web application or browser extension." applied.

Reply:
"This is a simpler option since it allows you to connect to your Ledger wallet via web application or browser extension." applied.